storage/s3.go

```diff
@@ -14,6 +15,8 @@ import (
 	"sync"
 	"time"
 
+	// "github.com/segmentio/encoding/json"
```

Comment:
https://github.com/peak/s5cmd/pull/300#discussion_r654318052

Please remove this line and run `go mod tidy && go mod vendor`. `go.mod` looks stale.